handler/category-dimension-import.go

```diff
@@ -44,7 +52,7 @@ func (h *CategoryDimensionImport) getSubmittedInstance(ctx context.Context, e *e
 	// validate instance state
 	if i.State != dataset.StateSubmitted.String() {
 		return i, "", &Error{
-			err:     errors.New("instance is in wrong state, no dimensions options will be imported"),
+			err:     errors.New("instance is in wrong state, no more dimensions options will be imported"),
```

Comment:
question:
if the instance is in the wrong state and no more dimensions will be imported ... 
How does the situation get resolved ?

Reply:
It's not resolved. I think in this case the import should be retried.
This case would happen, for example, if a previous handle failed (set the instance state to failed) - if it already failed, there would be no point in trying to handle new dimensions.